### DISCUSSION TITLE
Give ui image a set max-height

### DISCUSSION THREAD

**Discussion by Bradley-H** [fomantic/Fomantic-UI#1108] [Views (card, comment, item, statistic,...)] [answered]
So, right now with my cards' image not perfectly aligned without a max height. Currently height is set to auto which give undesirable results.

![Screenshot at 2019-10-19 06-44-08](https://user-images.githubusercontent.com/37282160/67132608-e3c76a80-f23b-11e9-9d27-941ae4b4fe09.png)

I think it would be more fitting to give the images (small, mini, medium...) image sizes a fixed max-height and max-width rather than width being <something>px and height being auto.

**Comment by lubber-de** [comment, marked as answer]
Could you please provide a jsfiddle with your above result? Did you use `ui cards` as a wrapper around your single cards?
Usually cards should have the same height and the image adjusts the overall cards height, because the card width is fixed.
See https://jsfiddle.net/1z2oL6c9/
Otherwise image sizes are currently ignored in cards to make sure the images fits the width. 
![image](https://user-images.githubusercontent.com/18379884/67146768-8ad4f200-f28e-11e9-864a-d16d26dd840a.png)

**Comment by y0hami** [comment]
@Bradley-H Is this still affecting you, if it is could you please provide a JSFiddle as @lubber-de has said above.

**Comment by lubber-de** [comment]
Closing because of no response. Will reopen when a reproducable jsfiddle is provided
